docs/source/getting_started.rst

```diff
@@ -33,6 +33,16 @@ Then, you can test your installation by executing:
 
 If the command executes without returning an error, you have successfully installed the Genestack Python Client Library. Yay!
 
+
+.. note::
+
+  If you have ssh warnings in console you can update to newest version of python 2.7.* or install requests security dependencies.
```

Comment:
If you see a warning such as ``InsecurePlatformWarning: A true SSLContext object is not available`` in the console, you can either update your Python to the latest ``2.7.*`` version, or install the ``security`` package extras using ``pip``: